examples/prompty/eval-apology/apology.prompty

```diff
@@ -0,0 +1,43 @@
+---
+name: Apology Prompt
+description: A prompt that determines whether a chat conversation contains an apology from the assistant
+model:
+  api: chat
+  configuration:
+    type: azure_openai
+    connection: open_ai_connection
+    azure_deployment: gpt-35-turbo-0125
```

Comment:
Can we change this deployment to gpt-35-turbo ? This will align with chat-basic and make sure the chat-with-prompty nb run smoothly. If the intent is having another model to do evaluation, then we also need another kind of model instead of another gpt-35-turbo

Reply:
This is because it requires certain version of gpt-35 to have the response_format: { "type": "json_object" } feature